config/initializers/application_controller_renderer.rb

```diff
@@ -0,0 +1,8 @@
+# Be sure to restart your server when you modify this file.
```

Comment:
Is this a Rails 5.2 default? Are we keeping it?

Reply:
And same for all the other initializers that are commented out

Reply:
I've chosen to revert all of the config files generated by rails back to what they were, with only the specific changes we make to them applied. For example, I'm not deleting all of the comments that come with them. This makes it so much easier to run diffs when upgrading to a new version. As it was, I had to pick through every line in our configs to map them to the 5.2 versions.